public/includes/addevent.php

```diff
@@ -0,0 +1,111 @@
+<?php
+session_start();
+if (!isset($_SESSION["uid"]))
+	header('location:/public/includes/login.php');
+?>	
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Add Event</title>
+	<link rel="stylesheet" type="text/css" href="../static/css/reset.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular.min.js"></script>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+	<link rel="stylesheet" href="../static/css/addgame.css">
+	<link rel="stylesheet" href="../static/css/footer.css">
+	<?php
+	if ($_SERVER["REQUEST_METHOD"]=== "POST")
+	{
+		function prepare($data){
+
+			$data = trim($data);
+			$data = stripslashes($data);
+			$data = htmlspecialchars($data);
+			return $data;
+		}
+		$gamename = prepare($_POST["gamename"]);
```

Comment:
you decide the name of image file with which it will be stored through the gamename that the user provides right?. why aren't there checks on this file name? one should never trust the user for correct input. what if the user inputs a file name with a bunch of '../../''s? think about it.